packages/mux-audio/test/web-test-runner.config.js

```diff
@@ -0,0 +1,16 @@
+import { esbuildPlugin } from "@web/dev-server-esbuild";
```

Comment:
is it possible to have a single config at the top level and then reference it in that location? Since these config files seem to be all the same.

Reply:
I'd say we should investigate that as a followup to not scope creep this PR, but I like the idea of in general trying to share more of our dev configs (tsconfig, esbuild, wtr, etc.)